src/gui/shell.cpp

```diff
@@ -835,6 +838,18 @@ void Shell::handleNeovimNotification(const QByteArray &name, const QVariantList&
 			} else {
 				emit neovimFullScreen(variant_not_zero(args.at(1)));
 			}
+		} else if (guiEvName == "WindowFrameless" && args.size() == 2) {
```

Comment:
Please move this block into a small function such as `handleWindowFrameless`.

You should be able to follow a similar pattern to `handleLineSpace` below.

Reply:
done